src/subscriptions/__tests__/subscriptionsReducer-test.js

```diff
@@ -108,6 +108,7 @@ describe('subscriptionsReducer', () => {
         subscriptions: [
           {
             name: 'some stream',
+            color: 'red',
```

Comment:
It does this in a way that doesn't look intentional; it may just be
    a consequence of not being forced to use well-typed (and therefore
    more likely to be representative) data in tests. The `color`

My reading is that it was probably intentional -- the two subscription objects looked different so that the test could confirm that the reducer kept the old one rather than putting in the new one. Partly that's my reading because the description says "do not add them", and if it replaced the old object with the new one then that'd be a way of "adding" the new one. So this seems like a natural thing to try to check if you're trying to write a test to be "thorough", rather than thinking about what the expected interface ought to be and sticking to that.

Reply:
Huh, interesting, I guess that makes sense. I had read "do not add them" as the negation of "add them" in the sense it's used in the other `op: 'add'` test: "(do not) concatenate items onto the state". But my reasoning turned out to be unhelpful because "add them" quite plausibly means "add or replace them" when there are partially matching items in the state already—and this isn't possible in the "add them" test, so it won't be addressed by negating "add them" in that sense.